packages/forms/backend/sirius-components-widget-reference/src/main/java/org/eclipse/sirius/components/widget/reference/ReferenceElementProps.java

```diff
@@ -51,10 +52,16 @@ public final class ReferenceElementProps implements IProps {
 
     private Setting setting;
 
+    private ReferenceWidgetStyle style;
+
     private ReferenceElementProps() {
         // Prevent instantiation
     }
 
+    public static Builder newMultiValuedReferenceElementProps(String id) {
```

Comment:
`newReferenceElementProps` (I know, the wrong name is not introduced by this patch, but if we can fix it while we're changing this class...).